test/integ_tests/test_tracking.py

```diff
@@ -0,0 +1,70 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+"""Tests that device resource tracking are correctly computed in the plugin device"""
+
+import numpy as np
+import pennylane as qml
+import pytest
+
+from braket.pennylane_plugin.braket_device import MIN_SIMULATOR_BILLED_MS, BraketAwsQubitDevice
+
+
+@pytest.mark.parametrize("shots", [100])
+class TestDeviceTracking:
+    """Tests for the device resource tracking"""
+
+    def test_tracking(self, device, shots, tol):
+        """Tests that a Device Tracker example correctly records resource usage"""
+
+        dev = device(1)
```

Comment:
Sorry, I don't know which device would be instantiated as a result of device(1) ? Also, is there a more expressive way to specify this ?

Reply:
This test will run with the local simulators and Braket's SV1 as specified in `devices` in `conftest.py`. I can add a comment stating this is the case. All of the existing integration tests are constructed this way, so I think it would be best to leave it specified this way be consistent.